src/plugins/abrt-action-analyze-c.c

```diff
@@ -259,7 +259,7 @@ int main(int argc, char **argv)
         sr_normalize_core_thread(thread);
 
         struct sr_core_frame *frame = thread->frames;
-        if (frame->function_name)
+        if (frame && frame->function_name)
```

Comment:
What if we also printed a message if we have no frames? That would help with future debugging.